astropy/coordinates/angle_utilities.py

```diff
@@ -173,6 +184,13 @@ def p_sign(p):
             else:
                 p[0] = 1.0
 
+        def p_dir(p):
+            '''
+            dir : DIRECTION
+            '''
+
+            p[0] = float(p[1])
```

Comment:
If you define `multiplier` above as a float rather than a string this can be `p[0] = p[1]` (just as for `sign`)